lib/memberManager.ts

```diff
@@ -149,16 +164,23 @@ export class Manager {
 
     logger("%d members", members.length);
 
-    const promises: Promise<Member | undefined>[] = [];
+    const promises: Promise<Member | undefined | void>[] = [];
 
     for (let member of members) {
       if (!this.state.has(member.value)) {
-        const promise = this.extractMember(member, page.data).then((member) => {
-          if (member) {
-            notifier.extracted(member, state);
-          }
-          return member;
-        });
+        const promise = this.extractMember(member, page.data)
+          .then((member) => {
+            if (member) {
+              notifier.extracted(member, state);
+            }
+            return member;
+          })
+          .catch((ex) => {
+            notifier.error(
+              { error: ex, type: "extract", memberId: member },
+              state,
+            );
+          });
 
         promises.push(promise);
       }
```

Comment:
The https://github.com/rdf-connect/ldes-client/pull/24/files#diff-da7e57eee5774b6f1726c34a35836992bfb7e13ff97057f20d9699901b542171L167 Promises.all() should have a .catch()